CHANGELOG.md

```diff
@@ -7,10 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```

Comment:
It might be worthwhile to add stub categories (Added, Changed, Deprecated, Removed, Fixed, Security) under Unreleased for people to tuck their message into.

Reply:
Added in e132a77.